src/client/components/ShoppingItem/ShoppingItem.js

```diff
@@ -10,32 +10,47 @@ export default function ShoppingItem({
   productName,
   quantity,
   price,
+  initValue,
+  isDisable,
 }) {
   const [itemValue, setItemValue] = useState(1);
```

Comment:
please initialize the state with initValue
const [itemValue, setItemValue] = useState(initValue);

Reply:
This is not necessary. We are taking value from initvalue already. This 1 is there as a puppet.

Reply:
it is necessary, because, when you load the page first time, then cost (item*quantity) does not get update on the page.. 
it only update when you press + or - on quantity

Reply:
I will raise another PR.. as shoppingItem component has to expose the cost of the item